client/ayon_core/pipeline/create/context.py

```diff
@@ -752,13 +753,12 @@ def _reset_creator_plugins(self) -> None:
         manual_creators = {}
         report = discover_creator_plugins(return_report=True)
         self.creator_discover_result = report
-        for creator_class in report.plugins:
-            if inspect.isabstract(creator_class):
-                self.log.debug(
-                    "Skipping abstract Creator {}".format(str(creator_class))
-                )
-                continue
+        for creator_class in report.abstract_plugins:
+            self.log.debug(
+                "Skipping abstract Creator '%s'", str(creator_class)
+            )
 
+        for creator_class in report.plugins:
```

Comment:
```python
            if not creator.product_base_type:
                warn(
                    f"Provided creator {creator!r} doesn't have"
                    " product base type attribute defined. This will be"
                    " required in future.",
                    DeprecationWarning,
                    stacklevel=2
                )
```
EDITED:
This should be after create plugin initialization before storing it by identifier (I can't put suggestion there).